CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing
+
+This document <https://github.com/fluxcd/community/blob/main/CONTRIBUTING.md> defines the contribution process for the Flux project and community.
+These guidelines apply to all git repositories in the `fluxcd` GitHub org, however each repository may specify additional contribution processes.
+
+## Welcome
+
+We gratefully welcome all kinds of contributions, including code, issues, documentation, external tools, advocacy and community work.
+All members of the Flux community, including contribors, are expected to uphold the Flux community [Values](https://github.com/fluxcd/community/blob/main/GOVERNANCE.md#values) and [Code of Conduct](https://github.com/fluxcd/community/blob/main/GOVERNANCE.md#code-of-conduct).
+
+## Communication
+
+As a contributor we want to invite you to join the discussions in a variety of forums laid out in <https://github.com/fluxcd/community>.
```

Comment:
> As a contributor we want to invite you to join the discussions in a variety of forums laid out in <https://github.com/fluxcd/community>.

Currently it doesn't - maybe we should just copy and paste https://github.com/fluxcd/flux2/blame/main/README.md#L88-L99 to the /community README.md for now?

Reply:
cf https://github.com/fluxcd/community/pull/30 - once that's merged, we could add `#contributing` to the link above?

Reply:
Yes 👍  agreed. I can add that to this PR as well

Reply:
added